frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -65,6 +68,21 @@ export const AddCapacityModal = withHandlePromise((props: AddCapacityModalProps)
     );
   }
 
+  const handleStorageClass = (sc: string, provisioner: string) => {
+    setStorageClass(sc);
+
+    if (_.isEqual(provisioner, 'kubernetes.io/no-provisioner')) {
+      setNoProvisionerSC(true);
+      getSCAvailablePVs(sc)
+        .then((pvs) => {
+          setPVsAvailableCapacity(humanizeBinaryBytes(calcPVsCapacity(pvs)).string);
+        })
+        .catch(() => setPVsAvailableCapacity('0 TiB'));
```

Comment:
if we are not able to get the capacity then ideally it should be not available

Reply:
ack